x-pack/plugins/rollup/public/crud_app/store/actions/create_job.js

```diff
@@ -33,34 +36,46 @@ export const createJob = (jobConfig) => async (dispatch) => {
       new Promise(resolve => setTimeout(resolve, 500)),
     ]);
   } catch (error) {
-    const { statusCode, data } = error;
+    if (error) {
+      const { statusCode, data } = error;
 
-    // Some errors have statusCode directly available but some are under a data property.
-    switch (statusCode || data.statusCode) {
-      case 409:
-        dispatch({
-          type: CREATE_JOB_FAILURE,
-          payload: {
-            error: {
-              message: `A job with ID '${jobConfig.id}' already exists.`,
+      // Expect an error in the shape provided by Angular's $http service.
+      if (data) {
+        // Some errors have statusCode directly available but some are under a data property.
+        if ((statusCode || (data && data.statusCode)) === 409) {
+          return dispatch({
+            type: CREATE_JOB_FAILURE,
+            payload: {
+              error: {
+                message: i18n.translate('xpack.rollupJobs.api.errors.createJobAlreadyExistsError.text', {
+                  defaultMessage: 'A job with ID \'{jobConfigId}\' already exists.',
```

Comment:
To prevent shielding of a quote you can wrap a string to back ticks.
```suggestion
                  defaultMessage: `A job with ID '{jobConfigId}' already exists.`,
```